app/src/main/res/layout/user_edit_view.xml

```diff
@@ -17,31 +19,31 @@
         <android.support.design.widget.TextInputLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            app:bindError="@{viewModel.firstNameError}">
+            bind:error="@{viewModel.firstNameError}">
 
             <android.support.design.widget.TextInputEditText
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:hint="@string/first_name"
-                app:bindTextChanges="@{viewModel::setFirstNameChanged}" />
+                bind:text="@={viewModel.firstName}"/>
```

Comment:
My problem with Inverse bindings is when both the view and view model can change the property, however if only 1 side is changing the property during the binding, it works. 

Will explain in the view model.